app/tests/functional/features/steps/exchange.py

```diff
@@ -7,7 +7,7 @@
 
 @given('I create an exchange')
 def given_create_an_exchange(context: Context):
-    r = requests.post('https://dev.api.bda.ninja/exchanges',
+    r = requests.post('https://dev.api.filetransfer.beta.octo.tools/exchanges',
```

Comment:
Maybe we can put the domain name `https://dev.api.filetransfer.beta.octo.tools` in a variable as we use it in multiple test cases

Reply:
Totally. I am wondering if func tests improvements should be in a dedicated PR?